stream-video-android-compose/src/main/kotlin/io/getstream/video/android/compose/ui/components/call/renderer/ParticipantsLayout.kt

```diff
@@ -37,13 +50,15 @@ import io.getstream.video.android.mock.mockCall
  * @param call The call that contains all the participants state and tracks.
  * @param modifier Modifier for styling.
  * @param style Defined properties for styling a single video call track.
+ * @param layoutType The type of layout. [LayoutType], default - [LayoutType.DYNAMIC]
  * @param videoRenderer A single video renderer renders each individual participant.
  */
 @Composable
-public fun ParticipantsGrid(
+public fun ParticipantsLayout(
```

Comment:
It would be great if we can also update relevant documentation:

https://github.com/GetStream/stream-video-android/blob/develop/docusaurus/docs/Android/04-ui-components/05-participants/02-participants-grid.mdx